src/legacy/ui/public/vislib/visualizations/_labels.scss

```diff
@@ -0,0 +1,22 @@
+$visBarLabelDarkColor: $euiColorDarkShade;
+$visBarLabelLightColor: $euiColorLightShade;
+
+.bar-label--normal {
```

Comment:
This can be simplified by using a base class of `.visColumnChart__barLabel` always with these two properties and then append `.visColumnChart__barLabel--stack` as the modifier class. Same with the color classes.